docs/Makefile

```diff
@@ -1,22 +1,23 @@
 # You can set these variables from the command line.
-POETRY        := $(HOME)/.poetry/bin/poetry
-SPHINXBUILD   := $(POETRY) run sphinx-build
-SPHINXOPTS    :=
-PAPER         :=
-BUILDDIR      := _build
-SOURCEDIR     := source
+POETRY ?=$(HOME)/.poetry/bin/poetry
+SPHINXBUILD ?=$(POETRY) run sphinx-build
+SPHINXOPTS ?=
+PAPER ?=a4
+BUILDDIR ?=_build
+SOURCEDIR ?=source
+OS ?=
 
 # For Windows users
 ifeq ($(OS),Windows_NT)
-    POETRY = $(APPDATA)\Python\Scripts\poetry
+    POETRY =$(APPDATA)\Python\Scripts\poetry
 endif
 
 # Internal variables.
-PAPEROPT_a4     := -D latex_paper_size=a4
-PAPEROPT_letter := -D latex_paper_size=letter
-ALLSPHINXOPTS   := -d $(BUILDDIR)/doctrees $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) $(SOURCEDIR) -W
+PAPEROPT_a4 ?=-D latex_paper_size=a4
```

Comment:
This section is described as `Internal variables`. I don't think overwriting them should be allowed?

Reply:
you can still overwrite then even with `:=` if you do `make PAPEROPT_a4=foo`. internals should be lowercase and there are some options to enforce it but I don't feel it's worth the effort. I can put these back though.